lints/lint_dnsname_underscore_in_trd.go

```diff
@@ -34,7 +34,7 @@ func (l *DNSNameUnderscoreInTRD) Execute(c *x509.Certificate) *LintResult {
 	if c.Subject.CommonName != "" {
 		underscoreFound, err := underscoreInTRD(c.Subject.CommonName)
 		if err != nil {
-			return &LintResult{Status: Fatal}
```

Comment:
We need to update to use the new helper function here as well.

Reply:
What's up with this?